docs/site/index.md

```diff
@@ -15,14 +15,15 @@ title: Unicode CLDR Project
 
 ## What is CLDR?
 
-The Unicode Common Locale Data Repository (CLDR) provides key building blocks for software to support the world's languages, with the largest and most extensive standard repository of locale data available. This data is used by a [wide spectrum of companies](#who-uses-cldr) for their software internationalization and localization, adapting software to the conventions of different languages for such common software tasks. It includes:
+The Unicode Common Locale Data Repository (CLDR) provides key building blocks for software to support the world's languages, with the largest and most extensive standard repository of locale data available. This data is supplied by contributors for their languages via the [CLDR SurveyTool](#translations-and-other-language-data).
```

Comment:
I think the comma after "languages" should be removed, but it's not new to this PR.

Reply:
Fixed just so that we don't have to remember to fix it later :)